src/ONE.Models.CSharp/ONE.Models.CSharp.csproj

```diff
@@ -18,9 +18,6 @@
     <PackageLicenseExpression>MIT</PackageLicenseExpression>
     <PackageRequireLicenseAcceptance>True</PackageRequireLicenseAcceptance>
     <PackageReleaseNotes>
-	Version 4.28.0
-	- Added Operation Export Protobuf
-	- Added LastLoginDateUTC On User Protobuf
```

Comment:
Is it ok to leave the comment out?

Reply:
We no longer add the comments to the csproj file. We instead add them to the CHANGELOG.md file so that it's more compatible with Github and contains a full history.

Reply:
I usually do add the comments for the latest version here but it is duplicated in the CHANGELOG as Kent said